src/elements/content-sidebar/AddTaskMenu.js

```diff
@@ -28,7 +28,11 @@ const AddTaskMenu = (props: Props) => (
             </MenuToggle>
         </Button>
         <Menu className="bcs-AddTaskMenu">
-            <MenuItem className="bcs-AddTaskMenu-menuItem" onClick={() => props.onMenuItemClick(TASK_TYPE_GENERAL)}>
+            <MenuItem
+                className="bcs-AddTaskMenu-menuItem"
+                data-target-id="MenuItem-generalTask"
```

Comment:
In a previous [PR](https://github.com/box/box-ui-elements/pull/3592/files) the part of the id following the - was camelcase but the first letter was capitalized. Shouldn't we be consistent?

Reply:
Wi stick to how it's named automatically by Babel plugin, example: https://git.dev.box.net/Hubs/hubs-client/pull/944/files
@bfoxx1906